src/content/runtime-apis/scheduled-event.md

```diff
@@ -0,0 +1,42 @@
+# ScheduledEvent
+
+## Background
+
+The event type for Scheduled requests to a Worker. The `Object` passed through as `event` in 
+[`addEventListener()`](/runtime-apis/add-event-listener).
```

Comment:
i think we may need to update the content on this linked page to indicate that `FetchEvent` isn't the only available type now

Reply:
Done.